README.md

```diff
@@ -0,0 +1,46 @@
+[![][ButlerImage]][wiki] 
+
+# About
+This is the source repository for the **File System SCM** plugin for Jenkins.
+This plugin provides an additional SCM option to Jenkins, based on the file
+system.
+
+For more information see the [wiki].
```

Comment:
Wiki is an outdated source of info. I would rather use https://plugins.jenkins.io/filesystem_scm.

You can also use markdowns like `[site](https://plugins.jenkins.io/filesystem_scm)` to keep unique links in-text

Reply:
Sure. Updated the PR.